pkg/resolver/client/ens/errors.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package ens
+
+import "fmt"
+
+// ErrNotImplemented denotes that the function has not been implemented.
+type ErrNotImplemented struct{}
+
+// Error returns the fomatted not implemented error message.
+func (e ErrNotImplemented) Error() string {
+	return "function not implemented"
+}
```

Comment:
This error type is pretty static. Could it be var ErrNotImplemented = errors.New("function not implemented")?